benchmarks/struct_indices/spider/evaluate.py

```diff
@@ -0,0 +1,291 @@
+import argparse
+import ast
+import json
+import logging
+import os
+from typing import Dict, List
+
+from langchain import OpenAI
+from langchain.chat_models import ChatOpenAI
+from langchain.schema import HumanMessage
+from spider_utils import create_indexes, load_examples
+from tqdm import tqdm
+
+from gpt_index.indices.struct_store.sql import GPTSQLStructStoreIndex
+
+logging.getLogger("root").setLevel(logging.WARNING)
+
+answer_template = (
```

Comment:
@jerryjliu Might be nice to start pulling these into the main LlamaIndex modules.

Reply:
yes agreed! expand the sql prompts we have to adapt to chatgpt